addons/wowl/static/src/components/error_dialogs/error_dialogs.ts

```diff
@@ -33,30 +33,21 @@ export class ErrorDialog extends Component<ErrorDialogProps, OdooEnv> {
 
   constructor() {
     super(...arguments);
-    const { data, message, stack: errorStack, subType, type } = this.props.error;
+    const { data, message, subType, traceback, type } = this.props.error;
     if (type === "server") {
       this.title = capitalize(subType) || this.env._t("Odoo Error");
     } else {
       this.title = this.env._t("Odoo Client Error");
     }
-    let stack = (data ? data.debug : errorStack) || "";
-    /**
-     * Error.prototype.stack is non-standard.
-     * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error
-     * However, most engines provide an implementation.
-     * In particular, Chrome formats the contents of Error.stack
-     * https://v8.dev/docs/stack-trace-api#compatibility
-     */
-    /** @todo add some key concerning the browser type in env, somewhere? */
-    // if (self.browserDetection.isBrowserChrome()) {  --> add to OdooBrowser? We should check things since now Edge uses chromium do they have same behavior?
-    //  traceback = ev.reason.stack;
-    // } else {
-    this.traceback = `${message}\n${stack}`;
-    // }
+    if (data) {
```

Comment:
I would leave in the comment on the non-standard nature of error stack somewhere (especially the refs to the docs).
And it's not because its mine :D
It's easier in the future to backtrack a decision.